Worker.Extensions.Sql/src/Microsoft.Azure.Functions.Worker.Extensions.Sql.csproj

```diff
@@ -7,11 +7,12 @@
 		<Product>SQL Binding Worker</Product>
 		<!--Version information-->
 		<VersionPrefix>99.99.99</VersionPrefix>
+		<SupportedVersion>1.*-*</SupportedVersion>
```

Comment:
This should be 99.99.99 to match the local dev package version

(and it would be even better to make a top level prop that is used by both this and the extension project)

Reply:
Well, this goes into the whole "testing locally should use the local version of packages" concept. So it shouldn't be trying to pull down from nuget.org - that's the whole point. And the fact that it does that currently shows that we're not using the correct version of the extension when running our tests which is problematic. 

So what we should be aiming for here is to have : 

1. The supported version be 99.99.99 when built locally (overridden in pipelines with whatever the new version is)
2. When you run using the local worker it is able to pick up the local nuget package. This will mean the tests need to know how to load the local version of the package (which likely means a nuget.config pointing to the bin folder as a source)

Reply:
Looking into adding a nuget config and make the tests use the local version of the package. Can I get this in for the pipeline variable changes? Or close this out and address all of them together in one PR?

Reply:
Let's do it as a follow up item - I suspect it'll be a bit more annoying and no reason to delay this since we're not regressing anything. Could you make an issue to track getting that done?